src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class ConsoleHandler {
+
+    @Inject
+    protected static UserDao userDao;
+    @Inject
+    protected static BetDao betDao;
+
+    public static void input() {
+        try (BufferedReader bReader = new BufferedReader(new InputStreamReader(System.in))) {
```

Comment:
bReader - isn't good name of the variable

Reply:
done)